womtool/src/test/resources/validate/wdl_draft2/invalid/http_import_missing_input/http_import_missing_input.wdl

```diff
@@ -0,0 +1,14 @@
+import "https://raw.githubusercontent.com/openwdl/wdl/0636dc21f2418faf301ae56571c1669fa947accc/versions/draft-2/parsers/javascript/tests/cases/0/wdl" as imports
```

Comment:
Could you put an `error.txt` next to this WDL to make sure we're getting the "bad inputs" error instead of the "bad import" error?